### DISCUSSION TITLE
Configurable fall damage

### DISCUSSION THREAD

**Discussion by emielderckx** [PurpurMC/Purpur#1137] [__Completed__] [answered]
Make the fall damage a player or mob takes configurable for:
- [ ] Hay bales (Assassin's Creed lol)
- [ ] Honey block
- [ ] Slime blocks

If possible also for
- [ ] Lava
- [ ] Water
- [ ] Beds
- [ ] Cobwebs
- [ ] Sweet berry bushes
- [ ] Powder snow

It seems there are no plugins that make this possible currently.
Idea original inspired by: https://modrinth.com/mod/softer-hay-bales

**Comment by granny** [comment, marked as answer]
This has been implemented since build 1812 https://github.com/PurpurMC/Purpur/commit/f28d282780d257e39838c082a6ab7a8914d9c65a (all the way back in version 1.19.2)
